packages/server-node/src/launch/socket-server-launcher.ts

```diff
@@ -30,11 +30,24 @@ export class SocketServerLauncher extends GLSPServerLauncher<net.TcpSocketConnec
 
     run(opts: net.TcpSocketConnectOpts): Promise<void> {
         const netServer = net.createServer(socket => this.createClientConnection(socket));
+
         netServer.listen(opts.port, opts.host);
-        this.logger.info(`The GLSP server is ready to accept new client requests on port: ${opts.port}`);
-        // Print a message to the output stream that indicates that the start is completed.
-        // This indicates to the client that the sever process is ready (in an embedded scenario).
-        console.log(this.startupCompleteMessage);
+        netServer.on('listening', () => {
+            const addressInfo = netServer.address();
+            // eslint-disable-next-line no-null/no-null
+            if (addressInfo === null) {
+                this.logger.error('Could not resolve GLSP Server address info');
+                return;
+            } else if (typeof addressInfo === 'string') {
+                this.logger.error('Unexpected type for AddressInfo');
```

Comment:
This should log more detailed information: 
```suggestion
                this.logger.error(`GLSP Server is unexpectedly listening to pipe or domain socket "${addressInfo}". Shutting down.`);
```

Reply:
Thanks @sdirix, I updated the log info with details